web/satellite/src/types/segment.ts

```diff
@@ -0,0 +1,10 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+import Vue from 'vue';
+
+declare module 'vue/types/vue' {
+    interface Vue {
+        $segment: any; // define real typings here if you want
```

Comment:
I'm wondering if rather than doing this which doesn't help to lint/check the code, we could use https://www.npmjs.com/package/@types/segment-analytics